js-w1-hw.js

```diff
@@ -0,0 +1,100 @@
+// Age-ify (A future age calculator) TASK 1
+
+const yearOfBirth = 1986;
+const yearFuture = 2069;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}.`);
+
+// Goodboy-Oldboy (A dog age calculator) TASK 2
+
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2030;
+
+const shouldShowResultInDogYears = true; // Change this to false to show the result in human years
+if (shouldShowResultInDogYears) {
+  dogYear = dogYearFuture - dogYearOfBirth; // Calculating the age in dog years
+  console.log(`Your dog will be ${dogYear} dog years old in ${dogYearFuture}.`); // Showing the result in dog years
+} else {
+  dogYear = (dogYearFuture - dogYearOfBirth) * 7; // Calculating the age in human years (assuming thet 1 human year equals to 7 dog years)
```

Comment:
You can define dogYear outside of the `if else`, as otherwise you repeat `dogYearFuture - dogYearOfBirth` twice.
For example, you could do this before the `if else`.
```
let dogYear = dogYearFuture - dogYearOfBirth;
```
For the case when the conditional logic is true, you would add before the `console.log` this:
```
  dogYear = dogYear * 7;
```